multilateralfund/urls.py

```diff
@@ -20,5 +20,5 @@
 from django.urls import path
 
 urlpatterns = [
-    path('admin/', admin.site.urls),
+    path('api/admin/', admin.site.urls),
```

Comment:
It will be a bit confusing to have the Django admin nested under "/api/`, I would suggest removing that.